### ISSUE TITLE
Huge regression in Mainline 1296 causes massive slowdown in all games

### ISSUE THREAD

**Issue by PlatinumLucario** [yuzu-emu/yuzu#9586]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1296, Mainline 1298, Mainline 1299

### Description of Issue

I've looked everywhere for any issue relating to this, however I've noticed every game now runs at a really small 1-2 frames per second, since Mainline 1296, which was the same time that the new Vulkan options were added into Yuzu.

Even despite disabling those new Vulkan options, every game still runs at 1-2 fps. This doesn't happen on Mainline 1295 or earlier, and I'm not sure if this occurs on graphics cards other than NVIDIA GeForce GTX 950. And if it does, please let us know!

This has also been tested on nvidia-driver-510, as well as nvidia-driver-525, both produce the same result.

If this is isolated to it only happening on specific graphic cards, then I could speculate that it might have something to do with a Tegra SoC function that doesn't exist on certain GPUs, but does exist on more modern GPUs. I could very well be wrong, and I hope to be wrong about it.

If the devs are all fully aware of this issue and are working on it, this can be closed.

### Expected Behavior

Normally, all games on Yuzu run near to their full frame rate, like it does with Mainline 1295 and earlier builds.

### Reproduction Steps

1. Open Yuzu.
2. Play any of your dumped games with any configuration.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10368459/yuzu_log.txt)

### System Configuration

CPU: Intel® Core™ i3-6100 CPU @ 3.70GHz × 4
GPU: NVIDIA Corporation GM206 [GeForce GTX 950]
GPU Driver: nvidia-driver-525
RAM: 16 GiB
OS: Ubuntu 22.04.1 LTS

**Comment by piplup55** [comment]
can you try and disable force maximum clocks
![image](https://user-images.githubusercontent.com/100526773/211197969-95ebac15-c089-412f-abf1-de89dd5208ab.png)

**Comment by PlatinumLucario** [comment]
> can you try and disable force maximum clocks ![image](https://user-images.githubusercontent.com/100526773/211197969-95ebac15-c089-412f-abf1-de89dd5208ab.png)

As I mentioned in the post, even disabling both Force maximum clocks and Use Vulkan pipeline cache, or even having just one of them on, won't make any difference.

I can confirm this with another log I just made, as well as a screenshot:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10368502/yuzu_log.txt)

![Screenshot from 2023-01-09 00-21-25](https://user-images.githubusercontent.com/1906189/211198426-c0e16298-fdb8-4fd0-92e2-6cdda8ad9a58.png)

And hey, if it has to come down to ending support for Maxwell GPUs on Yuzu, I can accept that, because it is a really old GPU that probably lacks some important GPU functions that the Tegra X1 SoC uses.

**Comment by DiegoMadGlez** [comment]
> > can you try and disable force maximum clocks ![image](https://user-images.githubusercontent.com/100526773/211197969-95ebac15-c089-412f-abf1-de89dd5208ab.png)
> 
> As I mentioned in the post, even disabling both Force maximum clocks and Use Vulkan pipeline cache, or even having just one of them on, won't make any difference.
> 
> I can confirm this with another log I just made, as well as a screenshot: [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10368502/yuzu_log.txt)
> 
> ![Screenshot from 2023-01-09 00-21-25](https://user-images.githubusercontent.com/1906189/211198426-c0e16298-fdb8-4fd0-92e2-6cdda8ad9a58.png)
> 
> And hey, if it has to come down to ending support for Maxwell GPUs on Yuzu, I can accept that, because it is a really old GPU that probably lacks some important GPU functions that the Tegra X1 SoC uses.

Any reason for running the game in single core?

**Comment by PlatinumLucario** [comment]
> Any reason for running the game in single core?
Absolutely! I was testing it in single core to make sure that it wasn't just multi-core that was affected. And I can confirm it occurs on both single core and multi-core options. Still slows down to a crawl.

Here's a fresh log of it in multi-core:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10368843/yuzu_log.txt)

**Comment by liamwhite** [comment]
This is because you are using extreme accuracy and single core mode. Switch to high.

**Comment by Phoenomenal** [comment]
gpu extreme accuracy is the reason as iamwhite write above.  I am replaying Xenoblade Chronicles definitive edition at the moment and its the same showing 1-2 fps (Xenoblade 2 same thing)   In previous yuzu versions (prior to 1296)   i was getting better fps with Extreme than with High  in Xenoblade games actually.  But now its completely broken.   So strange....

**Comment by PlatinumLucario** [comment]
> This is because you are using extreme accuracy and single core mode. Switch to high.

> gpu extreme accuracy is the reason as iamwhite write above. I am replaying Xenoblade Chronicles definitive edition at the moment and its the same showing 1-2 fps with gpu accurace set to extreme (Xenoblade 2 same thing) In previous yuzu versions (prior to 1296) i was getting better fps with Extreme than with High in Xenoblade games actually. But now its completely broken. So strange....

So I did some testing with it on High accuracy, which works really well, and while in-game, switching it back to Extreme(very slow), and turns out by switching it back to Extreme while in-game, I got it to crash instantly. Interesting...
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10369642/yuzu_log.txt)

I will need to update the title to "Regression in Extreme CPU accuracy in Mainline 1296 causes massive slowdown in all games".

**Comment by PlatinumLucario** [comment]
Okay, it seems to only occur in some games, more specifically, games developed by Monolithsoft (with the exception of The Legend of Zelda: BotW, which doesn't seem to have this issue).

**Comment by FernandoS27** [comment]
> > This is because you are using extreme accuracy and single core mode. Switch to high.
> 
> > gpu extreme accuracy is the reason as iamwhite write above. I am replaying Xenoblade Chronicles definitive edition at the moment and its the same showing 1-2 fps with gpu accurace set to extreme (Xenoblade 2 same thing) In previous yuzu versions (prior to 1296) i was getting better fps with Extreme than with High in Xenoblade games actually. But now its completely broken. So strange....
> 
> So I did some testing with it on High accuracy, which works really well, and while in-game, switching it back to Extreme(very slow), and turns out by switching it back to Extreme while in-game, I got it to crash instantly. Interesting... [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10369642/yuzu_log.txt)
> 
> I will need to update the title to "Regression in Extreme CPU accuracy in Mainline 1296 causes massive slowdown in all games".

yeah, you shouldn't switch back to Extreme while in-game.

**Comment by Phoenomenal** [comment]
> Okay, it seems to only occur in some games, more specifically, games developed by Monolithsoft (with the exception of The Legend of Zelda: BotW, which doesn't seem to have this issue).

It seems you right it only affects  some games.  Maybe even only Xenoblade series.  I tested about 6-7 games tonight and all of them run  excellent with gpu set to Extreme.      Well Xenoblade games is probably considered the hardest to emulate (judging from yuzu progress reports they are most constantly mentioned) .   And they are maybe the only games which had better fps on Extreme prior (on 60 fps mod i was getting on extreme setting  60 fps most of the time but on high 40-50),  but now broken on that setting.   I think you could add in Reproduction Steps  that Xenoblade games are 100% affected for devs to test them.

**Comment by PlatinumLucario** [comment]
I'm also really curious as to which compiled code in hex in the Xenoblade games causes this? Could it be limiters? Could it be a function? Method? Class? Variable?

I'd really love to look into this further, but I'm completely inexperienced with Switch hardware registers.

**Comment by PlatinumLucario** [comment]
Extreme GPU accuracy still slows to a crawl in Xenoblade games as of Mainline 1307. I will continue testing newer builds in case anything changes.

**Comment by Phoenomenal** [comment]
> Extreme GPU accuracy still slows to a crawl in Xenoblade games as of Mainline 1307. I will continue testing newer builds in case anything changes.

Good,  thanks.   I will monitor this too and will write when (if)  this will be fixed. Its pretty simple to me  since i am replaying Xenoblade DE right now  so game starts each time with extreme and then i change to high  when game session starts with 1-2 fps )

**Comment by Antivan28** [comment]
Isuue is fixed as of  Yuzu Mainline 1324.   Solid 30 fps gameplay with GPU set to Extreme.  Was really getting fps drops while on high  in  previous 3 weeks in Xenoblade DE (on I9 9900K and rtx 2080ti).  Still cant understand why Xenoblade games works so much better on Extreme option which supossed to be slow.....  But well thats it,  back to smooth gameplay in Xenoblade which is great.

**Comment by Phoenomenal** [comment]
Isuue is fixed as of Yuzu Mainline 1324. Solid 30 fps gameplay with GPU set to Extreme. Was really getting fps drops while on high in previous 3 weeks in Xenoblade DE (on I9 9900K and rtx 2080ti). Still cant understand why Xenoblade games works so much better on Extreme option which supossed to be slow..... But well thats it, back to smooth gameplay in Xenoblade which is great.
